engine/apps/api/tests/test_openapi_schema.py

```diff
@@ -1,13 +1,14 @@
 import pytest
 import yaml
+from django.test import override_settings
 from django.urls import reverse
 from rest_framework import status
 from rest_framework.test import APIClient
 
 
 @pytest.mark.django_db
+@override_settings(DRF_SPECTACULAR_ENABLED=True)
 def test_fetching_the_openapi_schema_works(settings, reload_urls):
-    settings.DRF_SPECTACULAR_ENABLED = True
```

Comment:
seeing if _hopefully_ this will fix the tests failing on Drone ([build](https://drone.grafana.net/grafana/oncall-private/6738/1/10))